server/server/Models/item.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const itemSchema = new mongoose.Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: true,
+    },
+    price: {
+      type: Number,
+      required: true,
+    },
+    itemPicture: {
```

Comment:
itemPicture is fine if it's not required, because we can put a placeholder image in frontend if image field is absent